### PR TITLE
Medical Vitals - Add event to vitals loop

### PR DESCRIPTION
**Pull request by pterolatypus**
**When merged this pull request will:**
- Add a local event in the main vitals loop so that extension mods can run updates in-sync without having to write their own loop.

### IMPORTANT

- [x] If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [x] [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/medical_vitals/functions/fnc_handleUnitVitals.sqf [hunk 1]
```diff
@@ -165,4 +165,7 @@ if (!isPlayer _unit) then {
 
 END_COUNTER(Vitals);
 
+//placed outside the counter as 3rd-party code may be called from this event
+[QEGVAR(medical,handleUnitVitals), _unit] call CBA_fnc_localEvent;
```
Comment (mharis001) [inline addons/medical_vitals/functions/fnc_handleUnitVitals.sqf:169]: Thoughts on providing `_deltaT` here as well for convenience?

Comment (pterolatypus) [reply addons/medical_vitals/functions/fnc_handleUnitVitals.sqf]: good call

**Comment by pterolatypus** [comment]
I feel like this should be documented, but I'm not sure where to put it as there isn't really any doc for vitals yet.

**Review by PabstMirror — APPROVED** [review]

**Comment by jonpas** [comment]
@pterolatypus Can put it on the Events page, where there is a list of API events.

**Pull request merged**